Sources/UInt128.swift

```diff
@@ -66,7 +66,7 @@ public struct UInt128 {
     }
     
     /// Undocumented private variable required for passing this type
-    /// to a FloatingPoint type. See FloatingPointTypes.swift.gyb in
+    /// to a BinaryFloatingPoint type. See BinaryFloatingPoint.swift.gyb in
```

Comment:
Can you change this to refer to the previously referenced file `FloatingPointTypes.swift.gyb`, while retaining the change referring to the `BinaryFloatingPoint` type? I've just taken a look at the latest commit on the swift repo (swift/swift@c6ec50e2bbd1b6dc2e37b8c488bf6f8424e0e6e4) and they have kept these definitions in the same file.